sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1578 @@
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# set other parameters\n",
+    "FORECAST_HORIZON = 12\n",
+    "TIME_SERIES_ID_COLNAMES = []\n",
+    "BLOCKED_MODELS = [\"ExtremeRandomTrees\"]"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1684500024383
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# difference data and test for unit root\n",
+    "if DIFFERENCE_SERIES:\n",
+    "    df_delta = df.copy()\n",
+    "    df_delta[TARGET_COLNAME] = df[TARGET_COLNAME].diff()\n",
+    "    df_delta.dropna(axis=0, inplace=True)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1684500026457
+    }
+   }
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "source": [
+    "### Split the data into train and test"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "from helper_functions import ts_train_test_split\n",
+    "\n",
+    "# split the data into train and test set\n",
+    "if DIFFERENCE_SERIES:\n",
+    "    # generate train/inference sets using data in first differences\n",
+    "    df_train, df_test = ts_train_test_split(\n",
+    "        df_input=df_delta,\n",
+    "        n=FORECAST_HORIZON,\n",
+    "        time_colname=TIME_COLNAME,\n",
+    "        ts_id_colnames=TIME_SERIES_ID_COLNAMES,\n",
+    "    )\n",
+    "else:\n",
+    "    df_train, df_test = ts_train_test_split(\n",
+    "        df_input=df,\n",
+    "        n=FORECAST_HORIZON,\n",
+    "        time_colname=TIME_COLNAME,\n",
+    "        ts_id_colnames=TIME_SERIES_ID_COLNAMES,\n",
+    "    )"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1684500028916
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "# Save the DataFrame objects to files\n",
+    "train_data_path = \"./data/train_S4248SM144SCEN.csv\"\n",
+    "test_data_path = \"./data/test_S4248SM144SCEN.csv\"\n",
+    "df_train.to_csv(train_data_path, index=False)\n",
+    "df_test.to_csv(test_data_path, index=False)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "gather": {
+     "logged": 1684500038680
+    }
+   }
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "source": [
+    "### Create the Azure Machine Learning MLTable"
+   ],
+   "metadata": {}
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "source": [
+    "With Azure Machine Learning MLTables you can keep a single copy of data in your storage, easily access data during model training, share data and collaborate with other users. \n",
+    "Below, we will upload the data by creating an MLTable to be used for training.\n",
+    "\n",
+    "**NOTE:** In this PRIVATE PREVIEW we're defining the MLTable in a separate folder and .YAML file.\n",
```

Comment:
Is this a private preview? If yes, we should consider making the same statement in the other notebooks.

Reply:
Removed. Thank you.